client/packages/invoices/src/Returns/modals/OutboundReturn/OutboundReturn.tsx

```diff
@@ -85,21 +74,20 @@ export const OutboundReturnEditModal = ({
         width={1024}
       >
         <>
-          <WizardStepper activeStep={getActiveStep()} steps={returnsSteps} />
-          <TabContext value={currentTab}>
-            <TabPanel value={Tabs.Quantity}>
-              <QuantityToReturnTable
-                lines={lines}
-                updateLine={line => update(line)}
-              />
-            </TabPanel>
-            <TabPanel value={Tabs.Reason}>
-              <ReturnReasonsTable
-                lines={lines}
-                updateLine={line => update(line)}
-              />
-            </TabPanel>
-          </TabContext>
+          {returnId && (
```

Comment:
There's a couple of conditionals which I don't love (hide/show the item selector, use the insert or update mutation...) - but otherwise basically identical between creating a return from inbound shipment lines, and adding an item to an existing return. Keen for feedback here!

Reply:
Seems reasonable to me 👍🏼